core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -201,6 +189,53 @@ public void testMetaGraphMaxRels() {
         db.executeTransactionally("MATCH (n) DETACH DELETE n");
     }
 
+    @Test
+    public void testRelationshipExistsSampling() {
+        List<Long> skipSizes = List.of(2L, 100L, 1000L, -1L, 0L, 1L);
+        List<Long> fromNodeCounts = List.of(4L, 1000L, 100L, 500L, 10000L, 4L);
+        List<Integer> expectedChecks = List.of(2, 10, 0, 500, 10000, 4);
```

Comment:
It would be nice if we could also test the impact of maxRels, either in this unit test or in the acceptance tests below

Reply:
Discussed, there are already such tests